tests/unit/beacons/test_sensehat_beacon.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+
+# Python libs
+from __future__ import absolute_import
+
+# Salt testing libs
+from tests.support.unit import skipIf, TestCase
+from tests.support.mock import NO_MOCK, NO_MOCK_REASON, patch, MagicMock
+from tests.support.mixins import LoaderModuleMockMixin
+
+# Salt libs
+import salt.beacons.sensehat as sensehat
```

Comment:
Avoid creating module level mocks, it will delay the garbage collection and any changes made to them will leak from test to test.
Create them in the setup modules func, assign to self and then `self.addCleanup(delattr, nickname)`